src/test/scala/uk/gov/nationalarchives/consignmentexport/ValidatorSpec.scala

```diff
@@ -66,10 +66,10 @@ class ValidatorSpec extends ExportSpec {
     val validator = Validator(UUID.randomUUID())
     val fileId = UUID.randomUUID()
     val metadata = FileMetadata("ClientSideOriginalFilepath", "filePath") :: Nil
-    val ffidMetadata = FfidMetadata("software", "softwareVersion", "binaryVersion", "containerVersion", "method", List(Matches("ext".some, "id", "puid".some)))
-    val files = List(Files(fileId,"File".some,"name".some, None, metadata, ffidMetadata.some, Option.empty))
+    val ffidMetadata = FfidMetadata("software", "softwareVersion", "binaryVersion", "containerVersion", "method", List(Matches("ext".some, "id", "puid".some, false.some, "formatName".some)))
+    val files = List(Files(fileId, "File".some, "name".some, None, metadata, ffidMetadata.some, Option.empty))
     val result = validator.extractFFIDMetadata(files)
-    val expectedResult = ValidatedFFIDMetadata("filePath", "ext", "puid", "software", "softwareVersion", "binaryVersion", "containerVersion")
+    val expectedResult = ValidatedFFIDMetadata("filePath", "ext", "puid", "formatName", extensionMismatch = false, "software", "softwareVersion", "binaryVersion", "containerVersion")
```

Comment:
Don't need to specifically have the param name for `extensionMismatch` in the constructor argument as all arguments are being used in the constructor.